lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
+      'sessionTokenVerified',
```

Comment:
Aha, found it in the code - I believe the string you're thinking of is the "tokenTypeID":

  https://github.com/mozilla/fxa-auth-server/blob/master/lib/tokens/session_token.js#L28

We can safely have multiple auth strategies that uses tokens of the same type.